src/python/py/models/builder.py

```diff
@@ -298,9 +298,15 @@ def make_genai_config(self, model_name_or_path, extra_kwargs, out_dir):
             "past_key_names": "past_key_values.%d.key",
             "past_value_names": "past_key_values.%d.value",
         })
+        if self.model_type.find("ChatGLM") != -1:
+            # config.bos_token_id not present in ChatGLM model configs.
+            bos_token_id = 1
+        else:
+            bos_token_id = config.bos_token_id
+
         genai_config = {
             "model": {
-                "bos_token_id": config.bos_token_id,
+                "bos_token_id": bos_token_id,
```

Comment:
Can we simplify this change into one line and say `"bos_token_id": config.bos_token_id if hasattr(config, "bos_token_id") else 1`?

Reply:
sure